ev-dev-tools/src/ev_cli/type_parsing.py

```diff
@@ -166,13 +168,13 @@ def generate_type_headers(cls, type_with_namespace, all_types, output_dir):
 
         types_parts = {'types': None}
 
-        output_path = output_dir / type_with_namespace['relative_path']
+        output_path = output_dir / type_with_namespace['rel_path']
         types_file = output_path.with_suffix('.hpp')
         output_path = output_path.parent
         output_path.mkdir(parents=True, exist_ok=True)
 
         namespaces = ['types']
-        namespaces.extend(type_with_namespace["relative_path"].parts)
```

Comment:
I know it's already been this way, but we should probably make the use of '' or "" more consistent within this python package

Reply:
Since I found significantly more occurrences of apostrophes (''), I changed all occurences of quotation marks ("") to apostrophes.